.github/workflows/luis_ci.yaml

```diff
@@ -41,13 +41,13 @@ jobs:
         git config remote.origin.url https://x-access-token:${{ secrets.GITHUB_TOKEN }}@github.com/${{ github.repository }}
         git fetch --prune --unshallow    
 
-    - name: Install GitVersion
-      uses: gittools/actions/gitversion/setup@v0.9.3
+    - name: Install GitVersion v0.9.3
+      uses: gittools/actions/gitversion/setup@6dfe406
```

Comment:
You may want to call out in the YAML or in `pipeline.md` that this is a commit hash rather than a version and we use that for security reasons

Reply:
Maybe TMI 🙂

Reply:
I think TMI...